core/typedefs.h

```diff
@@ -174,6 +174,9 @@ inline void __swap_tmpl(T &x, T &y) {
 
 static _FORCE_INLINE_ unsigned int next_power_of_2(unsigned int x) {
 
+	if (x == 0)
+		return 0;
```

Comment:
Braces will be enforced in the near future, see #33027

Reply:
Yeah but this will be done by a script. No need to refactor existing code to match it yet (it's not bad to do it either, but it's not necessary).